tests/retention/dump.cc

```diff
@@ -80,7 +80,7 @@ TEST_F(RetentionDumpTest, DumpHost) {
   std::ostringstream oss;
   dump::hosts(oss);
   std::string str(oss.str());
-  ASSERT_EQ(str, "");
+  ASSERT_EQ(str, "host {\nhost_name=test_host1\nhost_id=12\nacknowledgement_type=0\nactive_checks_enabled=1\ncheck_command=\ncheck_execution_time=0.000\ncheck_flapping_recovery_notification=0\ncheck_latency=0.000\ncheck_options=0\ncheck_period=\ncheck_type=0\ncurrent_attempt=1\ncurrent_event_id=0\ncurrent_notification_id=0\ncurrent_notification_number=0\ncurrent_problem_id=0\ncurrent_state=0\nevent_handler=\nevent_handler_enabled=1\nflap_detection_enabled=1\nhas_been_checked=0\nis_flapping=0\nlast_acknowledgement=0\nlast_check=0\nlast_event_id=0\nlast_hard_state=0\nlast_hard_state_change=0\nlast_notification=0\nlast_problem_id=0\nlast_state=0\nlast_state_change=0\nlast_time_down=0\nlast_time_unreachable=0\nlast_time_up=0\nlong_plugin_output=\nmax_attempts=3\nmodified_attributes=0\nnext_check=0\nnormal_check_interval=5.000\nnotification_period=\nnotifications_enabled=1\nnotified_on_down=0\nnotified_on_unreachable=0\nobsess_over_host=1\npassive_checks_enabled=1\npercent_state_change=0.00\nperformance_data=\nplugin_output=\nproblem_has_been_acknowledged=0\nprocess_performance_data=1\nretry_check_interval=5.00\nstate_type=1\nrecovery_been_sent=0\nstate_history=0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0\n}\n");
```

Comment:
This test may fail. All depends if you only launch this test or all.
I have commented the compilation of dump.cc because dump tests should be done when a host is created.
This test is not removed just to keep the way to do it.

Reply:
ok does I leave it as is ?